Event Management - Updating docs for GA (#22586)
DataDog/documentation: content/en/service_management/events/correlation/triage_and_notify.md

1. Click into any of the alerts or events, and we automaically pull in all related metrics and logs, so that you can view them in context of the alert
1. (Optional) Select any alerts or events you want to remove that are not related to the case. Click the event to open a side panel detailing related logs and metrics. Troubleshoot issues with all the related data pulled into one page.
1. Under the *Related Metrics* section, compare all related metrics or group by tags.

## Create a notification or ticket

With correlated events, you can configure one notification for a group. So, instead of having 20 notifications and 20 potential issues to investigate, you have one single case and one notification. Combine all your alerts in the Case Management Projects page. There are a few ways to group notifications in Case Management:

**Ticketing**

On the Project Settings page, configure the Integrations you want your projects to send notifications to. Today we support both **ServiceNow** and **Jira** manual and automatic ticket creating with bi-directional syncing. For setup instructions, see the [Case Management Settings][3] documentation.

**Notifications**

> Views in case management group cases based on a configured query. You can set up a notification when a case matching this query is created. Today we support, **Pagerduty**, **Email**, **Webhook**, **Microsoft Teams**, **Slack**. To learn how to create a view, see the [Case Management Views][4] documentation.

Reviewer: ```suggestion
In case management, _views_ group cases based on a configured query. You can set up a notification when a case matching this query is created. Datadog supports, **Pagerduty**, **Email**, **Webhook**, **Microsoft Teams**, and **Slack**. To learn how to create a view, see the [Case Management Views][4] documentation.
```
